docker/server/Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN pip3 install urllib3 cryptography --upgrade
 
 ADD . /tmp/
 RUN cd /tmp/ && pip3 install -r requirements.txt
+RUN cd /tmp/ && pip3 install -r dfvfs_requirements.txt
```

Comment:
why do we have dfvfs usage in the server?

Reply:
There really shouldn't be, but right now it unfortunately it gets loaded because when the job manager loads PartitionsEnumerationJob, it loads a different module that loads it.  Nothing actually uses it, and this should really get refactored out.